packages/vaadin-combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -0,0 +1,307 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { Virtualizer } from '@vaadin/vaadin-virtual-list/src/virtualizer.js';
+import { ComboBoxPlaceholder } from './vaadin-combo-box-placeholder.js';
+
+/**
+ * Element for internal use only.
+ *
+ * @extends HTMLElement
+ * @private
+ */
+class ComboBoxScrollerElement extends PolymerElement {
+  static get is() {
+    return 'vaadin-combo-box-scroller';
+  }
+
+  static get template() {
+    return html`
+      <style>
+        :host {
+          display: block;
+          min-height: 1px;
+          overflow: auto;
+
+          /* Fixes item background from getting on top of scrollbars on Safari */
+          transform: translate3d(0, 0, 0);
+
+          /* Enable momentum scrolling on iOS */
+          -webkit-overflow-scrolling: touch;
+
+          /* Fixes scrollbar disappearing when 'Show scroll bars: Always' enabled in Safari */
+          box-shadow: 0 0 0 white;
+        }
+
+        #selector {
+          border-width: var(--_vaadin-combo-box-items-container-border-width);
+          border-style: var(--_vaadin-combo-box-items-container-border-style);
+          border-color: var(--_vaadin-combo-box-items-container-border-color);
+        }
+      </style>
+      <div id="selector">
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  static get properties() {
+    return {
+      items: {
+        type: Array,
+        observer: '__itemsChanged'
+      },
+
+      focusedIndex: {
+        type: Number,
+        observer: '__focusedIndexChanged'
+      },
+
+      loading: {
+        type: Boolean,
+        observer: '__loadingChanged'
+      },
+
+      opened: {
+        type: Boolean
+      },
+
+      selectedItem: {
+        type: Object
+      },
+
+      itemIdPath: {
+        type: String
+      },
+
+      comboBox: {
+        type: Object
+      },
+
```

Comment:
This seems unused

Reply:
The `comboBox` reference is used by `vaadin-combo-box-item` component.

Reply:
Ok, so it seems